app/src/main/java/ch/epfl/sweng/favours/database/Favor.java

```diff
@@ -33,6 +36,7 @@ public void updateOnDb(){
                         documentID = docRef.getId();
                         updateFromDb();
                     }).addOnFailureListener(e -> {
+                        Log.d(TAG,"failure to push to database");
                 /* Feedback of an error here - Impossible to update user informations */
```

Comment:
launchToast("It is currently impossible to add the Favor. please check your connection");